src/features/user/Account/ReactivateModal.tsx

```diff
@@ -27,12 +25,11 @@ export const ReactivateModal = ({
   fetchRecurrentDonations,
 }: ReactivateModalProps) => {
   const [disabled, setDisabled] = React.useState(false);
-  const { tenantConfig } = useTenant();
+  const { putApiAuthenticated } = useApi();
   const { theme } = React.useContext(ThemeContext);
-  const { token, logoutUser } = useUserProps();
   const { setErrors } = React.useContext(ErrorHandlingContext);
   const t = useTranslations('Me');
-  const bodyToSend = {};
+  const payload = {};
```

Comment:
Why do we send an empty payload?

Reply:
1. The payload is not required for this request.
2. Since we have a strict type definition for PUT requests to accept a payload, we provide an empty object to avoid any type errors.

Reply:
I think we need to simplify the type structure defined in useApi.ts and apiClient.ts a bit. It has a few inconsistencies. But that is a separate matter.